tests/pom.xml

```diff
@@ -476,7 +476,7 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                       <exec>
                         <arg>/sbin/qdrouterd</arg>
                         <arg>-c</arg>
-                        <arg>/etc/hono/qdrouterd-with-broker.json</arg>
+                        <arg>/etc/hono/qdrouterd-with-artemis.json</arg>
```

Comment:
this refers to the config file that has been copied into the container from location `src/test/resources/qpid/qdrouter-with-broker.json`. So unless you have changed the name of that file (which I think you haven't), this should remain the way it was ... We do not want to run the integration tests against ServiceBus either, do we?
IMHO this is also the reason for the failing Travis build ...

Reply:
@sophokles73 ahhhh, that makes a lot of sense. Thank you!